src/libraries/System.IO.MemoryMappedFiles/src/System/IO/MemoryMappedFiles/MemoryMappedFileAcl.Unix.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO.MemoryMappedFiles
+{
+    public static class MemoryMappedFileAcl
```

Comment:
Would it be better to let these implementations to be auto-generated in System.IO.FileSystem.AccessControl with a better PlatformNotSupportedException?

Reply:
What's the command to auto generate the files?

Reply:
I don't think we have a way to auto-generate only a list of APIs, we can only auto-generate the not supported assemblies using GenAPI at compile time for all the APIs defined in the ref assembly. @ericstj can correct me if I'm wrong.